keyboards/minidox/keymaps/norman/keymap.c

```diff
@@ -0,0 +1,70 @@
+#include QMK_KEYBOARD_H
+
+#include "keymap_norman.h"
+
+#define PERMISSIVE_HOLD
+#define IGNORE_MOD_TAP_INTERRUPT
+#define QMK_KEYS_PER_SCAN 4
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _NORMAN 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 16
```

Comment:
Setting your `_ADJUST` layer to 16 when there are only four layers will increase your firmware size for no benefit.

```suggestion
#define _ADJUST 3
```

----

**OPTIONAL:**

Define your layers as an `enum` instead:

```c
enum layers {
    _NORMAN,
    _LOWER,
    _RAISE,
    _ADJUST
};
```